acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/UpCheckAcceptanceTest.java

```diff
@@ -15,30 +15,51 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import tech.pegasys.ethsigner.tests.dsl.http.HttpResponse;
+import tech.pegasys.ethsigner.tests.dsl.node.NodeConfiguration;
+import tech.pegasys.ethsigner.tests.dsl.node.NodeConfigurationBuilder;
+import tech.pegasys.ethsigner.tests.dsl.node.NodePorts;
+import tech.pegasys.ethsigner.tests.dsl.signer.Signer;
+import tech.pegasys.ethsigner.tests.dsl.signer.SignerConfiguration;
+import tech.pegasys.ethsigner.tests.dsl.signer.SignerConfigurationBuilder;
 
 import java.net.SocketTimeoutException;
 
 import io.netty.handler.codec.http.HttpResponseStatus;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 
-public class UpCheckAcceptanceTest extends AcceptanceTestBase {
+public class UpCheckAcceptanceTest {
 
   private static final String UP_CHECK_PATH = "/upcheck";
   private static final String UP_CHECK_MESSAGE = "I'm up!";
-  private static final String TIMEOUT_MESSAGE = "timeout";
+
+  private static Signer ethSigner;
+
+  @BeforeAll
+  public static void setUpBase() {
+    Runtime.getRuntime()
+        .addShutdownHook(new Thread(DataPathFeatureFlagAcceptanceTest::tearDownBase));
+
+    final NodeConfiguration nodeConfig = new NodeConfigurationBuilder().build();
+    final SignerConfiguration signerConfig =
+        new SignerConfigurationBuilder().withHttpRpcPort(0).withWebSocketPort(0).build();
+
+    ethSigner = new Signer(signerConfig, nodeConfig, new NodePorts(1, 2));
+    ethSigner.start();
+    ethSigner.awaitStartupCompletion();
+  }
 
```

Comment:
Similar to the other comment, this test is also missing a tearDownBase to cleanup.